### PR TITLE
Refresh when focused after opening in browser

### LINKED ISSUE #1: Refresh after opening notifications in browser (after a delay)
Another nice extension from you 🤣

After I view the notifications in the browser, the status bar item remains unchanged. I would suggest refreshing in a minute after "open in browser" command.

### PR DESCRIPTION
**Pull request by yzhang-gh**
Resolve #1.

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -12,10 +12,24 @@ function activate ( context: vscode.ExtensionContext ) {
...
+
   return statusbar.init ();
 
 }
 
+function onDidChangeWindowState ( event ) {
+
+  if (event.focused && Utils.state.get ( 'didOpenInBrowser', false ) ) {
+    
+    vscode.commands.executeCommand ( 'githubNotificationsBell.refresh', true );
```
Comment (fabiospampinato) [inline src/extension.ts]: Can you call the command directly instead of using `executeCommand`?

File: src/utils.ts [hunk 2]
```diff
@@ -19,7 +19,7 @@ const Utils = {
 
       const commandName = _.last ( command.split ( '.' ) ) as string,
             handler = Commands[commandName],
-            disposable = vscode.commands.registerCommand ( command, () => handler () );
+            disposable = vscode.commands.registerCommand ( command, args => handler ( args ) );
```
Comment (fabiospampinato) [inline src/utils.ts]: There's no need for this if you call the command directly.

File: src/commands.ts [hunk 3]
```diff
@@ -9,11 +9,13 @@ import statusbar from './statusbar';
 
 /* COMMANDS */
 
-async function refresh () {
+async function refresh ( silence? ) {
```
Comment (fabiospampinato) [inline src/commands.ts]: "silence" doesn't sound super descriptive to me, can you replace `silence?` with `showNotification = true` maybe?

File: src/extension.ts [hunk 4]
```diff
@@ -12,10 +12,24 @@ function activate ( context: vscode.ExtensionContext ) {
...
 
+  context.subscriptions.push ( vscode.window.onDidChangeWindowState ( onDidChangeWindowState ) );
+
   return statusbar.init ();
 
 }
 
+function onDidChangeWindowState ( event ) {
+
+  if (event.focused && Utils.state.get ( 'didOpenInBrowser', false ) ) {
```
Comment (fabiospampinato) [inline src/extension.ts:24]: This looks wrong, shouldn't we check if `didOpenInBrowser` is `true` instead?

**Review by fabiospampinato — CHANGES_REQUESTED** [review]

File: src/extension.ts (hunk 4 above)
Comment (yzhang-gh) [reply src/extension.ts:24]: It is checking whether `didOpenInBrowser` is `true` 😳

(I have tested it using fake notifications)

File: src/extension.ts (hunk 1 above)
Comment (yzhang-gh) [reply src/extension.ts]: Forgot that 😅

File: src/extension.ts (hunk 4 above)
Comment (fabiospampinato) [reply src/extension.ts:24]: Right, I misread that, sorry 😬

**Comment by fabiospampinato** [comment]
Published as v2.1.0. Thank you so much!

**Pull request merged**